src/GafferTest/ContextTest.cpp

```diff
@@ -204,3 +205,39 @@ void GafferTest::testEditableScope()
 	}
 
 }
+
+void GafferTest::testContextHashPerformance( int numEntries, int entrySize, bool startInitialized )
+{
+	// We usually deal with contexts that already have some base stuff in them
+	ContextPtr baseContext = new Context();
+	for( int i = 0; i < numEntries; i++ )
+	{
+		baseContext->set( std::to_string( i ), std::string( entrySize, 'x') );
```

Comment:
Probably not important here, but there's an `InternedString( int64_t )` constructor that should be faster than `InternedString( to_string( i ) )`.

Reply:
Oh, good point.

Reply:
Fixed in 1820604